LICENCE.rst

```diff
@@ -1,8 +1,11 @@
 GrantNav
 ========
 
-GrantNav is free software designed to help people check data 
-pubplished to the Open Contracting Data Standard.
+Note: This license doesn't apply to the 360Giving logo https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo.png https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo_invert.png
```

Comment:
I don't know, I would assume All Rights Reserved (c) 360 Giving
I'm not sure what we want to put anything in our license file about this, and whether we should clarify it with 360Giving.